packs/equipment/cursebreak-bulwark.json

```diff
@@ -10,7 +10,7 @@
         },
         "containerId": null,
         "description": {
-            "value": "<p>This tower shield is composed of interlocking hexagonal wooden tiles lined with metal. The tiles are painted a verdant green, and each is marked with a faded rune that protects against curses. While you have this shield raised, you gain a +1 circumstance bonus to saving throws against spells that target you, increasing to a +2 circumstance bonus against curses. You can unleash the energy stored in the shield to destroy a curse before it can affect you.</p>\n<p>AC increases to +4 if you are using the @UUID[Compendium.pf2e.actionspf2e.Item.Take Cover] action.</p>\n<hr />\n<p><strong>Activate</strong> <span class=\"action-glyph\">r</span> command</p>\n<p><strong>Frequency</strong> once per day</p>\n<p><strong>Trigger</strong> You are targeted with a curse spell</p>\n<p><strong>Requirements</strong> The cursebreak bulwark is raised</p>\n<hr />\n<p><strong>Effect</strong> You activate the shield's countermeasures. It casts 8th-rank @UUID[Compendium.pf2e.spells-srd.Item.Cleanse Affliction] with a counteract modifier of [[/r 1d20+27 #Counteract]]{+27} against the triggering curse.</p>\n<table class=\"pf2e\">\n<thead>\n<tr>\n<th>Hardness</th>\n<th>HP</th>\n<th>BT</th>\n</tr>\n</thead>\n<tbody>\n<tr>\n<td>14</td>\n<td>56</td>\n<td>28</td>\n</tr>\n</tbody>\n</table>"
+            "value": "<p>This tower shield is composed of interlocking hexagonal wooden tiles lined with metal. The tiles are painted a verdant green, and each is marked with a faded rune that protects against curses. While you have this shield raised, you gain a +1 circumstance bonus to saving throws against spells that target you, increasing to a +2 circumstance bonus against curses. You can unleash the energy stored in the shield to destroy a curse before it can affect you.</p><hr /><p><strong>Activate</strong> <span class=\"action-glyph\">r</span> command</p>\n<p><strong>Frequency</strong> once per day</p>\n<p><strong>Trigger</strong> You are targeted with a curse spell</p>\n<p><strong>Requirements</strong> The <em>cursebreak bulwark</em> is raised</p><hr /><p><strong>Effect</strong> You activate the shield's countermeasures. It casts 8th-rank @UUID[Compendium.pf2e.spells-srd.Item.Cleanse Affliction] with a counteract modifier of [[/r 1d20+27 #Counteract]]{+27} against the triggering curse.</p><table class=\"pf2e\"><thead><tr><th>Hardness</th><th>HP</th><th>BT</th></tr></thead><tbody><tr><td>14</td><td>56</td><td>28</td></tr></tbody></table>"
```

Comment:
Removed the "AC increases to +4 if you are using the Take Cover action.", as that's just part of the base item, and the current UI seems to convey that clearly enough.